app/angular/package.json

```diff
@@ -80,7 +80,7 @@
     "ts-loader": "^8.0.14",
     "tsconfig-paths-webpack-plugin": "^3.3.0",
     "util-deprecate": "^1.0.2",
-    "webpack": "4"
+    "webpack": ">=4.0.0 <6.0.0"
```

Comment:
React's is a bit different. Do we care?
```
    "webpack": ">=4.43.0 <6.0.0"
```

Reply:
didn't want to introduce new constraints